core/src/main/java/hudson/logging/LogRecorderManager.java

```diff
@@ -161,6 +163,19 @@ public HttpResponse doNewLogRecorder(@QueryParameter String name) {
         return new HttpRedirect(name + "/configure");
     }
 
+    public FormValidation doCheckNewName(@QueryParameter String name) {
```

Comment:
```suggestion
    @Restricted(NoExternalUse.class)
    public FormValidation doCheckNewName(@QueryParameter String name) {
```

(or even `DoNotUse`, but that's not imported yet).